app/models/user.rb

```diff
@@ -1,6 +1,14 @@
 class User < ApplicationRecord
+  include RecommendedFeedItemConcern
+
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_and_belongs_to_many :feeds, join_table: :user_feeds
+
+  def full_name
+    "#{first_name} #{last_name}"
+  end
 end
```

Comment:
Our user model is simple, allowing us to identify any feeds a particular user has recommended (starred); this comes in handy in our UI where we need to identify _who_ has recommended a particular feed item.